ex36 - without is_game_continue.py

```diff
@@ -0,0 +1,206 @@
+from sys import exit
+
+
+def start():
+	print("You don't remember anything. You are fallen.")

```

Comment:
Create a function, e.g. show_introduction(), for 3 prints

Reply:
Ok. How about I apply all the introduction lines of all functions (chest() , crow(), great_wall(), etc. ), into smaller print functions? Will it be clean code?

Reply:
No need for now. The function start() should be the most abstract. Other functions can have low level statements